core/src/main/java/com/novoda/merlin/MerlinBuilder.java

```diff
@@ -77,13 +79,18 @@ public MerlinBuilder withAllCallbacks() {
     }
 
     /**
-     * Sets the internal Merlin Logging state. Uses the TAG : Merlin
+     * Deprecated, use directly {@link Logger} instead. To provide backwards compatibility
+     * this method will clear the {@link Logger} and attach a new {@link MerlinBackwardsCompatibleLog}
+     * which will absorb or log depending on the withLogging passed.
      *
-     * @param withLogging by default Logging is disabled. withLogging = true will enable logging, withLogging = false will disable.
-     * @return MerlinBuilder.
+     * @param withLogging by default logging is disabled. withLogging = true will attach the default {@link MerlinBackwardsCompatibleLog}
+     * @return MerlinBuilder
      */
+    @Deprecated
     public MerlinBuilder withLogging(boolean withLogging) {
-        MerlinLog.LOGGING = withLogging;
+        Logger.detachAll();
```

Comment:
or it should be local state - e.g. instead of having a static logger, it should be a dependency, so that you can construct 2 merlin instances without affecting each other.